packages/preview-middleware/package.json

```diff
@@ -40,12 +42,14 @@
         "ejs": "3.1.7"
     },
     "devDependencies": {
+        "@private/preview-middleware-client": "workspace:@sap-ux-private/preview-middleware-client@*",
```

Comment:
It is the only why to prevent the `update-ts-references` script from adding it as reference to the `tsconfig` - which is wrong in this case because we don't build the client module with `tsc`.

Reply:
OK. Thanks for that info.